src/Lachain.Core/Blockchain/VM/ExternalHandler.cs

```diff
@@ -308,6 +308,14 @@ public static void Handle_Env_WriteEvent(int signatureOffset, int valueOffset, i
             };
             frame.InvocationContext.Snapshot.Events.AddEvent(ev);
         }
+        
+        public static void Handler_Env_GetAddress(int resultOffset)
+        {
+            var frame = VirtualMachine.ExecutionFrames.Peek() as WasmExecutionFrame
+                        ?? throw new InvalidOperationException("Cannot call KECCAK256 outside wasm frame");
```

Comment:
Perhaps "?? throw new InvalidOperationException("Cannot call GetAddress outside wasm frame");" is better?

Reply:
yes... will update the code.